index.html

```diff
@@ -303,6 +315,12 @@ <h4 class="modal-title" id="twelveStepLabel">TWELVE STEP PROGRAMS</span></h4>
 
             <h3><strong>Pennyrile Area (Paducah & Hopkinsville)</strong></h3>
             <p><a href="http://www.na-pana.org">http://www.na-pana.org</a></p>
+            
+            <h3><strong>Participation Station</strong></h3>
+            <p>869 Sparta Court, Lexington, KY<br>
+            Thursdays 12-1:00 p.m.<br>
+            Sunday 3-4:00 p.m.<br>
+            Phone: <a href="tel:8594926997">(859) 492-6997</a></p>
```

Comment:
all three of these locations are where one can find `Double Trouble in Recovery` programs.